src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -156,8 +156,8 @@
   <data name="CannotWritePropertyAfterProperty" xml:space="preserve">
     <value>Cannot write a JSON property name following another property name. A JSON value is missing.</value>
   </data>
-  <data name="CannotWriteValueAfterPrimitive" xml:space="preserve">
-    <value>Cannot write a JSON value after a single JSON value. Current token type is '{0}'.</value>
+  <data name="CannotWriteValueAfterPrimitiveOrClose" xml:space="preserve">
+    <value>Cannot write a JSON value after a single JSON value or outside of an existing closed object/array. Current token type is '{0}'.</value>
```

Comment:
I borrowed this message from `CannotStartObjectArrayAfterPrimitiveOrClose`:
https://github.com/dotnet/corefx/blob/00b49d189519677732bcde79a91bee955f3f7a82/src/System.Text.Json/src/Resources/Strings.resx#L132-L134

If we consider "a JSON value" as any complete JSON payload (whether a single primitive value, or something contained within `[ ... ]`, `{ ... }`), then maybe the "or outside ..." addition isn't necessary and we can simplify the exception message.

I would be interested in hearing what others think about it.